### DISCUSSION TITLE
Saved search/filters?

### DISCUSSION THREAD

**Discussion by MACscr** [rappasoft/laravel-livewire-tables#362] [General]
Are table states saved by default so that if i user leaves the table to view lets say one of its elements edit pages or view pages and then comes back to the table, will the existing filters/search still be there? I need this as it exists on the jquery datatables that was being used before the new version of our app, but doesnt appear to exist with MedicOneSystems/livewire-datatables and a quick search of this repo didn't seem to turn up anything either.

**Comment by rappasoft** [comment]
It is actually supposed to, it seems I may have broken it between 1.9 and 1.10. I'll get back to you.

**Reply by rappasoft** [reply]
Ok, I just patched it.

The query string saves the state of the table. For example, I can link you [here](https://tables.laravel-boilerplate.com/tailwind?filters%5Bsearch%5D=ang&filters%5Bactive%5D=yes&sorts%5Bemail%5D=asc&sorts%5Bname%5D=asc) and you should see:

<img width="1330" alt="Screen Shot 2021-06-22 at 00 07 29" src="https://user-images.githubusercontent.com/6026504/122861708-e2dabc00-d2ed-11eb-9125-b9b34227bba1.png">

So you can save the query string and redirect back to that when you are done.

**Comment by lrljoe** [comment]
Possible in v2+
